src/app/main-column.component.ts

```diff
@@ -22,248 +22,251 @@ declare let mwf: any;
+          this.updateVersionFromEndpointUrl();
+          this.updateGraphVersionSelect();
+
+          this.updateHttpMethod();
+
+          // Add content-type header when switching to POST
+          if ((this.oldExplorerValues.selectedOption !== 'POST' && this.explorerValues.selectedOption === 'POST')
+              || (this.oldExplorerValues.selectedOption !== 'PUT' && this.explorerValues.selectedOption === 'PUT')
+              || (this.oldExplorerValues.selectedOption !== 'PATCH' && this.explorerValues.selectedOption === 'PATCH')
+          ) {
+              // If it doesn't already exist
+              let hasContentTypeHeader = false;
+              if (this.explorerValues.headers) {
+                  for (const header of this.explorerValues.headers) {
+                      if (header.name.toLowerCase() === 'content-type') {
+                          hasContentTypeHeader = true;
+                          break;
+                      }
+                  }
+                  if (!hasContentTypeHeader) {
+                      this.explorerValues.headers.unshift({
+                          enabled: true,
+                          name: 'Content-type',
+                          readonly: false,
+                          value: 'application/json',
+                      });
+                  }
+              }
+          }
+
+          this.oldExplorerValues = JSON.parse(JSON.stringify(this.explorerValues));
+      }
+  }
+
+  public ngAfterViewChecked() {
+      /**
+       * Disable the the httpVerb picker after the view has changed and the client is not authenticated. We are doing
+       * like this since the httpVerb picker is a non-Angular, Microsoft Web Framework component that is loaded into
+       * the DOM. It is not part of the Angular template and is loaded at ngAfterViewInit().
+       */
+      if (this.isAuthenticated()) {
+          this._httpMethodEl.element.nativeElement.children[1].setAttribute('aria-disabled', 'false');
+          this._httpMethodEl.element.nativeElement.children[1].children[0].removeAttribute('disabled');
+      } else {
+          this._httpMethodEl.element.nativeElement.children[1].setAttribute('aria-disabled', 'true');
+          this._httpMethodEl.element.nativeElement.children[1].children[0].setAttribute('disabled', '');
+      }
+  }
+
+  public ngAfterViewInit(): void {
+      // Init httpMethod
+      mwf.ComponentFactory.create([{
+          component: mwf.Select,
+          elements: [this._httpMethodEl.element.nativeElement],
+          callback: (event: any) => {
+              this.updateHttpMethod();
+              event[0].selectMenu.subscribe({
+                  onSelectionChanged: (method) => {
+                      this.explorerValues.selectedOption = method.id;
+
+                      const { id: httpVerb } = method;
+                      if (httpVerb === 'GET') { this.showShareButton = true; }
```

Comment:
Can we remove `const { id: httpVerb } = method;` and make this:

`if (this.explorerValues.selectedOption === 'GET')`

Reply:
Also, are there any other changes? It is hard to see changes since there were spacing changes in the same commit.